rest-api/dao/physical_measurements_dao.py

```diff
@@ -298,8 +298,9 @@ def _update_participant_summary(self, session, obj):
     participant_summary.lastModified = clock.CLOCK.now()
 
     if obj.status and obj.status == PhysicalMeasurementsStatus.CANCELLED:
-      # update corresponding participant summary physicalMeasurementsStatus
-      participant_summary.physicalMeasurementsStatus = obj.status
+      if not self.existing_pm(session, participant):
```

Comment:
Shouldn't this condition be inverted? I don't follow

Reply:
no. its "if you do not have an existing pm that is valid, set this to cancelled"

Reply:
Got it now - maybe reword as `self.has_uncancelled_pm()`

Reply:
done